webpack.prod.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
```

Comment:
- [ ] Same case here. As directed in the previous reviewer, we need to reenable linters here and address the issue that was pointed by the linters. Please take care of this.